modules/renter/download.go

```diff
@@ -0,0 +1,150 @@
+package renter
+
+import (
+	"crypto/rand"
+	"errors"
+	"io"
+	"os"
+	"time"
+
+	"github.com/NebulousLabs/Sia/crypto"
+	"github.com/NebulousLabs/Sia/modules"
+)
+
+// A Download is a download that has been queued by the renter.
+type Download struct {
+	completed   bool
+	destination string
+	nickname    string
+
+	renter *Renter
+}
```

Comment:
okay. I think I'll put that on the list of 'to do eventually' things.

One thing though about not exporting them is that then external functions can't take advantage of any extra features they might offer, which could be useful for testing.

I guess 'extra features' kinda defeats the purpose of an interface.